app/v2/controllers/CreateHistoricNonFHLUkPiePeriodSummaryController.scala

```diff
@@ -115,4 +141,10 @@ class CreateHistoricNonFHLUkPiePeriodSummaryController @Inject()(val authService
       case ServiceUnavailableError | InternalError => InternalServerError(Json.toJson(errorWrapper))
       case _                                       => unhandledError(errorWrapper)
     }
+
+  private def auditSubmission(details: FlattenedGenericAuditDetail)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[AuditResult] = {
+    val event =
+      AuditEvent("CreateAndAmendHistoricFhlPropertyBusinessAnnualSubmission", "CreateAndAmendHistoricFhlPropertyBusinessAnnualSubmission", details)
```

Comment:
`CreateAndAmendHistoricFhlPropertyBusinessAnnualSubmission` I was wondering if this is a typo as the endpoint is `CreateAmendHistoricNonFHLUkPropertyIncomeExpensesPeriodSummary`?